.github/workflows/lint.yaml

```diff
@@ -16,11 +16,11 @@ jobs:
     timeout-minutes: 5
     steps:
     - uses: actions/checkout@v3
-    - uses: actions/setup-go@v3
+    - uses: actions/setup-go@v4
```

Comment:
Starting from actions/setup-go v4, the cache option is enabled by default, so we no longer need to explicitly specify `cache: true`.
https://github.com/actions/setup-go/releases/tag/v4.0.0